ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -436,24 +436,24 @@
 
 | Interface | Enabled |
 | --------- | ------- |
-{%         for ethernet_interface in err_cor_enc_intfs | arista.avd.natural_sort %}
-{%             if ethernet_interfaces[ethernet_interface].error_correction_encoding.enabled is arista.avd.defined(false) %}
+{%         for ethernet_interface in err_cor_enc_intfs | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```

Comment:
the conversion and sorting is not needed here, since you build the `err_cor_enc_intfs` list above in line 427-432. With these changes the `err_cor_enc_intfs` goes from being a list of keys to a list of dicts.